developer/system/gui.md

```diff
@@ -9,6 +9,11 @@ redirect_from:
 - /wiki/GUIdocs/
 ---
 
+<style>
+/* Override Bootstrap's nowrap (breaks table layout) */
+article table { white-space: normal; }
+</style>
+
```

Comment:
Instead of this, shouldn't we add to or edit the CSS files?

Reply:
Sounds good. Do we have a custom CSS file? I couldn't find it.

Reply:
see main repo: https://github.com/QubesOS/qubesos.github.io/

Reply:
Okay, I see now - the `white-space: nowrap` hack was added because of the admin API table (https://github.com/QubesOS/qubes-issues/issues/853). I'll try to do something about it, and for now I'm removing this fix from here.